ui/shared/utils/constants.js

```diff
@@ -389,14 +388,6 @@ export const INDIVIDUAL_EXPORT_DATA = [].concat(
   INDIVIDUAL_ID_EXPORT_DATA, INDIVIDUAL_CORE_EXPORT_DATA, [INDIVIDUAL_HAS_DATA_EXPORT_CONFIG], INDIVIDUAL_HPO_EXPORT_DATA,
 )
 
-export const familyVariantSamples = (family, individualsByGuid, samplesByGuid) => {
```

Comment:
as explained above with individuals, it is way faster to have one selector where everything is sorted, rather than sorting each sub-group individually